medusa/providers/torrent/html/morethantv.py

```diff
@@ -192,19 +193,29 @@ def login(self):
         if any(dict_from_cookiejar(self.session.cookies).values()):
             return True
 
+        # Get the login page, to retrieve the token
+        response = self.session.get(self.urls['login'])
+        token = re.search(r'token".value="([^"]+)"', response.text)
+        if not token:
+            log.warning('Unable to get login token')
+            return False
+
         login_params = {
             'username': self.username,
             'password': self.password,
-            'keeplogged': '1',
-            'login': 'Log in',
+            # screen_resolution (x) | screen_resolution (y) | color depth | timezone offset.
+            'cinfo': f'1536|864|24|{int(time.timezone / 60)}',
```

Comment:
I had to set the screen resolution and color depth to fixed values (that matched my laptop's screen).
So this might divert from what your sending if your manually accessing MTV from your computer. I don't suspect they will bann you for this. As you should be allowed to access your site with multiple devices from the same ip.